package.json

```diff
@@ -25,11 +25,11 @@
   },
   "homepage": "https://github.com/devongovett/browserify-istanbul",
   "dependencies": {
-    "istanbul": "~0.3.6",
     "minimatch": "^2.0.0",
     "through": "^2.3.4"
   },
   "devDependencies": {
+    "istanbul": "^0.4.2",
```

Comment:
That would be ideal, but `optionalDependencies` have so many problems that's it's more practical to exclude it altogether and add install step in the readme.

But in general yes, we're on the same page.

@devongovett Any objections/concerns/questions?